BlockingQueueSplit.tla

```diff
@@ -52,9 +52,11 @@ Get(t) ==
 
 -----------------------------------------------------------------------------
 
-TypeInv == /\ Len(buffer) \in 0..BufCapacity
-           /\ waitSetP \in SUBSET Producers
-           /\ waitSetC \in SUBSET Consumers
+TypeInv ==
+   /\ buffer \in Seq(Producers) 
```

Comment:
I eventually eliminated this conjunct because I didn't like how it makes the proof reason about the elements in the buffer, which, for modeling reasons, are even Producers.

Reply:
This is now in main.